requirements.in

```diff
@@ -16,10 +16,10 @@ itsdangerous==2.1.2
 jsonschema[format]==4.5.1
 marshmallow-sqlalchemy==0.28.1
 marshmallow==3.15.0
+M2Crypto==0.38.0
```

Comment:
Provides X509 standard cert validation for AWS .pem files (for validating SNS delivery receipt signatures). 20 years old and well-maintained.